src/uu/du/src/du.rs

```diff
@@ -720,6 +743,35 @@ Try '{} --help' for more information.",
     0
 }
 
+struct Threshold(Option<u64>, bool);
+
+impl FromStr for Threshold {
+    type Err = ParseSizeError;
+
+    fn from_str(s: &str) -> std::result::Result<Self, Self::Err> {
+        let offset = if s.starts_with('-') || s.starts_with('+') {
+            1
+        } else {
+            0
+        };
+        let sz = parse_size(&s[offset..])?;
+
+        Ok(Threshold(
+            Some(u64::try_from(sz).unwrap()),
+            s.starts_with('-'),
+        ))
+    }
+}
+
+impl Threshold {
+    fn should_exclude(&self, sz: u64) -> bool {
```

Comment:
I would prefer words as variable names, so `size` in this case.

Reply:
Will do.